waddrmgr/db.go

```diff
@@ -1865,3 +1816,165 @@ func upgradeToVersion3(namespace walletdb.Namespace, seed, privPassPhrase, pubPa
 	}
 	return nil
 }
+
+// upgradeToVersion4 upgrades the database from version 3 to version 4
+// The following fields were added to the serialized address row:
+// * 'watchingOnly' - boolean indicating whether the address is watching-only
+// This is required to flag each imported address as watching-only
+// * 'encryptedPubKeyHash' - encrypted pubkey hash of imported p2pkh address
+// This is required to import addresses without the associated public key.
+func upgradeToVersion4(namespace walletdb.Namespace) error {
+	err := namespace.Update(func(tx walletdb.Tx) error {
+		currentMgrVersion := uint32(4)
```

Comment:
No need for this local.  Just set it directly in the call to `putManagerVersion` below.